src/map/petskill.cpp

```diff
@@ -225,6 +225,7 @@ uint16 CPetSkill::getMsgForAction() const
     return getID();
 }
 
+// Converts skill's message id to the non-primary target version
 uint16 CPetSkill::getAoEMsg() const // TODO: put this in parent class?
```

Comment:
fwiw, this function was just lifted directly from CMobSkill so im not surprised it needed adjustment